nucleus/payara-modules/nucleus-microprofile/config-service/src/main/java/fish/payara/nucleus/microprofile/config/spi/ConfigProviderResolverImpl.java

```diff
@@ -300,6 +302,9 @@ Config getNamedConfig(String applicationName) {
             for (Properties props : getDeployedApplicationProperties(appName)) {
                 sources.add(new PropertiesConfigSource(props, appName));
             }
+            for (Properties props : getDeployedApplicationAliasProperties(appName)) {
+                sources.add(new AliasPropertiesConfigSource(props));
```

Comment:
I wonder why `PropertiesConfigSource` gets the `appName` passed while `AliasPropertiesConfigSource` does not.

Reply:
It's not actually used in either, but I left the original alone originally as I thought I might have missed something.
On second look though I can't see anything that uses it since it's private scoped with no getter - I'll remove it